src/main/kotlin/org/partiql/scribe/problems/ScribeProblem.kt

```diff
@@ -20,7 +20,9 @@ public class ScribeProblem(
             INTERNAL_ERROR -> "INTERNAL_ERROR"
             UNSUPPORTED_OPERATION -> "UNSUPPORTED_OPERATION"
             UNSUPPORTED_PLAN_TO_AST_CONVERSION -> "UNSUPPORTED_PLAN_TO_AST_CONVERSION"
+            UNSUPPORTED_AST_TO_TEXT_CONVERSION -> "UNSUPPORTED_AST_TO_TEXT_CONVERSION"
             INVALID_PLAN -> "INVALID_PLAN"
+            TRANSLATION_INFO -> "TRANSLATION_INFO"
```

Comment:
self-review: some additional problem codes and new problem severity `INFO` that will be used in subsequent PRs